app/main/views/frameworks.py

```diff
@@ -1193,22 +1193,4 @@ def view_contract_variation(framework_slug, variation_slug):
 @main.route('/frameworks/<framework_slug>/opportunities', methods=['GET'])
 @login_required
 def opportunities_dashboard(framework_slug):
```

Comment:
I wonder if it's worth renaming this view (with `_redirect` or `_deprecated`) to make sure it's not used to generate links within the apps anymore. All`url_for` calls should be using `external.opportunities_dashboard`